tests/tests_utils.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
```

Comment:
Renaming `tests/tests_utils.py` --> `tests/test_utils.py` allows pytest to find and run your tests with just: `pytest`
https://docs.pytest.org/en/6.2.x/goodpractices.html#test-discovery